### PR TITLE
fix(local debug telemetry): correct appid in telemetry

### PR DESCRIPTION
**Pull request by swatDong**
Update `getLocalTeamsAppId` due to config structure change.

### CODE REVIEW

**Comment by kuojianlu** [comment]
Could you please also correct [`getLocalTeamsAppId`](https://github.com/OfficeDev/TeamsFx/blob/dev/packages/cli/src/utils.ts#L270) in CLI preview?

File: packages/vscode-extension/src/debug/commonUtils.ts [hunk 1]
```diff
@@ -236,30 +236,48 @@ export async function getPortsInUse(): Promise<number[]> {
...
       const env = getActiveEnv();
       const envJsonPath = path.join(ws, `.${ConfigFolderName}/env.${env}.json`);
       const envJson = JSON.parse(fs.readFileSync(envJsonPath, "utf8"));
-      return envJson.solution.teamsAppTenantId;
+      const settingValue = jsonSelector(envJson) as string;
+      if (settingValue && settingValue.startsWith("{{") && settingValue.endsWith("}}")) {
+        // setting in env.xxx.json is place holder and need to get actual value from xxx.userdata
+        const placeHolder = settingValue.replace("{{", "").replace("}}", "");
+        const userdataPath = path.join(ws, `.${ConfigFolderName}/${env}.userdata`);
+        if (fs.existsSync(userdataPath)) {
```
Comment (kuojianlu) [inline packages/vscode-extension/src/debug/commonUtils.ts:253]: If `settingValue` is **{{xxx}}** `userdataPath` does not exists, the function will also return **{{xxx}}**, is it our expectation?

**Review by kuojianlu — APPROVED** [review]

**Comment by swatDong** [comment]
> Could you please also correct [`getLocalTeamsAppId`](https://github.com/OfficeDev/TeamsFx/blob/dev/packages/cli/src/utils.ts#L270) in CLI preview?

Good catch, updated

**Comment by swatDong** [comment]
@chagong @Alive-Fish @jiayi-ruan  - this PR fixes both vscode extension and cli to correct the local debug app id in telemetry.
Please help review as code owner of cli `util.cs`

**Review by chagong — APPROVED** [review]

**Pull request merged**